### PR TITLE
fix(component): cards divider line

### PR DESCRIPTION
**Pull request by gepifanio**
Ticket: [#312](https://app.zenhub.com/workspaces/ffam-prototype-5c615588801d5a103031639c/issues/unimelb/ffam-prototype/312)

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit ba2d6f8c33a1e0c70e4730afe06cfcb72479085b

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5cdd057608c338000973f131

File: components/cards/_card-generic.css [hunk 1]
```diff
@@ -36,8 +36,7 @@
   .card__sub-titles {
     margin-bottom: .25rem;
     border-top: 1px solid rgb(var(--col-black-mid-1));
-    border-bottom: 1px solid rgb(var(--col-black-mid-1));
-    padding: 16px 0;
+    padding-top: 16px;
```
Comment (TimMansell) [inline components/cards/_card-generic.css]: use rem instead

File: components/cards/_card-generic.css [hunk 2]
```diff
@@ -36,8 +36,7 @@
...
     font-weight: 600;
```
Comment (TimMansell) [inline components/cards/_card-generic.css]: Not part of the PR, but work using the variable for font-weight here.

File: components/cards/_card-generic.css [hunk 3]
```diff
@@ -61,6 +60,8 @@
   }
 
   .card__excerpt {
+    border-top: 1px solid rgb(var(--col-black-mid-1));
+    padding-top: 16px;
```
Comment (TimMansell) [inline components/cards/_card-generic.css]: use rem instead

File: components/cards/_card-generic.css [hunk 4]
```diff
@@ -36,8 +36,7 @@
   .card__sub-titles {
     margin-bottom: .25rem;
```
Comment (TimMansell) [inline components/cards/_card-generic.css]: This kind of seems pointless.  Works out to 4px.  Removing it doesn't change the layout too much.  I think we should remove it.
